pmg/utils.py

```diff
@@ -48,14 +48,15 @@ def track_pageview(path=None, ignore_bots=True):
         # GA1.2.1760224793.1424413995
         client_id = client_id.split(".", 2)[-1]
 
-    tracker = Tracker.create(ga_id, user_id=user_id, client_id=client_id)
-    tracker.send(
-        "pageview",
-        path,
-        uip=request.access_route[0],
-        referrer=request.referrer or "",
-        userAgent=user_agent,
-    )
+    with HTTPRequest() as http:
```

Comment:
shouldn't this be the request that triggered the call rather than a blank new request?

Reply:
I know it'll take a bit more wiring to pass the request along, but if I'm not mistaken, this will give the impression that every logged request is a new user, totally messing up the data.

How did it work before? Did it get the request from the context?

Reply:
TBH I didn't think about what this is actually for. The Python 2 [version](https://github.com/adswerve/universal-analytics-python) of this package didn't require the `http` parameter. I'll look into to find out what how it's used.

Reply:
Looks like the old package created a new request using the user agent if there was one:

https://github.com/adswerve/universal-analytics-python/blob/78ee36a5f1efc144043ae003c855b97fbe662f7c/UniversalAnalytics/Tracker.py#L217-L218

Reply:
Ah cool - ok, this isn't the time to be improving these things - so we're just maintaining behaviour. I'll take a peak at the analytics in a day or so just to be sure.